_data/open.yml

```diff
@@ -887,7 +910,7 @@ concepts:
       property: mixfix
       area: "set theory"
       comments:
-       - "<math><mi>f</mi><mo>:</mo><mi>X</mi><mo>⤖</mo><mi>Y</mi></math>"
+       - "<math><mrow intent='bijection($a1,$a2,$a3)'><mi arg='a1'>f</mi><mo>:</mo><mi arg='a2'>X</mi><mo>⤖</mo><mi arg='a3'>Y</mi></mrow></math>"
```

Comment:
For bijection the example with `f` was poorly chosen by me - predates properties. It would be `f:bijection` in a world where `is-a` property use was still available.

A better example would be an infix arrow, but those are non-standard, [math SE discussion](https://math.stackexchange.com/questions/46678/what-are-usual-notations-for-surjective-injective-and-bijective-functions). Maybe `X ↔ Y` with the `↔` marked as `intent="bijection"`, maybe not.

Reply:
Not sure why you say infix arrows are non standard, all the arrows are infix binary by default. Oh you mean people on that SE thread don't agree on which arrows to use, not that arrows being infix are non standard, OK.

But yes, in general lots of the examples will need to be reviewed there are several I've spotted while doing this round that I plan to raise issues on after the merge.

But I'm trying to keep this PR  just on the format so replacing `notation:` field with mathml hints to actual mathml. Current status is 1258 `<math>` elements and 248 `notation[a-z]:` fields so it's getting there...

But if someone (hint) would approve the merge I could merge this bulk edit to w3c, and then individual issues and/or PR  could be raised on problematic examples.

As it is, editing the w3c copy will lead to merge conflicts and only I have edit writes on my fork so I really would rather forks are short lived.

Reply:
The bijection change was made in this PR, hence I reviewed it here. If anything, I have been trying to keep my comments to a helpful minimum.

I will pass my review flag to Moritz @physikerwelt who - if interested - may help resolve the PR. I am not comfortable doing much more here.

Reply:
as you wish, I wouldn't have said it's a change here Ijust added mathml if we decide to treat this as a  property rather than a concept the whole row wiuld presumably go, but that's fine there will neeed to be several such edits whatever format the notation is shown. mathml with or without tintent attributes.

Reply:
My comments:
- Can we have intent attributes on the mathml root element, that way we would not need to add an mrow?
- I think the overall pattern a1-9 is good.
- Can I check the result of this PR somewhere visually?

Reply:
@physikerwelt  we could have intent om math but i added the mrow as if you ever want to coy these fragments into a larger expression  you would been to habve mrows so it seemed simpler to add them even though it does add one extra layer in these smal examples. Yoes the fork is rendering at  https://davidcarlisle.github.io/mathml-docs/intent-open-concepts/ including the button to show mathml source. (Currently you need to refresh the page to get back to the initial state)

Reply:
Thank you. This looks good. Two minor issues:

- Invalid markup: Incorrect number of children for <msup/> tag.
- “mathvariant='bold'” on MathML elements is deprecated and will be removed at a future date.

Reply:
Thank you Moritz for the time reviewing the PR. 

I would suggest to David to tag Moritz directly in future PRs as the outcomes appear more productive.